src/main/resources/db/migration/V1__initial-migration.sql

```diff
@@ -12,4 +14,14 @@ CREATE TABLE image
     created_at           DATE,
     source               VARCHAR(200) NOT NULL,
     active               BOOLEAN
-);
```

Comment:
When making changes while using a migration system, we shouldnt modify an existing migration, it should be a new migration. It can be V2__adding-lobby-table.sql

Reply:
Also, this is probably why the tests are failing